src/components/ReservationForm.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import api from '../redux/api';
+
+import '../styles/reservationForm.scss';
+
+const ReservationForm = () => {
+  const navigate = useNavigate();
+  const userObject = sessionStorage.getItem('logged_user');
+  const jsonObject = JSON.parse(userObject);
+  const dispatch = useDispatch();
+  const [unitId, setUnitId] = useState('');
+  const [location, setLocation] = useState('');
+  const [reserveDate, setReserveDate] = useState('');
+
+  const { list } = useSelector((state) => state.units);
+
+  useEffect(() => {
+    dispatch(api.fetchUnits());
+  }, [dispatch]);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    dispatch(
+      api.postReservation({
+        unit_id: unitId,
+        user_id: jsonObject.id,
+        location,
```

Comment:
[optional]
Location is never updated, and it doesn't reflect on the back-end Schema and should be removed